common/build-style/qbs.sh

```diff
@@ -0,0 +1,32 @@
+#
+# This helper is for templates using the Qt Build System (qbs)
+#
+do_configure() {
+	local qmake qt
+	if [ -x "/usr/lib/qt5/bin/qmake" ]; then
```

Comment:
i'm not a big fan of this approach since we can't guarantee that these files won't be brought up as dependencies.

I think it is better we separate into qbs-qt4 and qbs-qt5

Reply:
It does but we can't guarantee the right one will be picked.

With the current script if a dependency pulls in qt-qmake then qt4 will be used unconditionally.

I think it is better we have separate build_style and the developer knows which one to pick an has full confidence the right one will be used

Reply:
OK, done.